Applications/Resnet/tensorflow/transLayer.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
```

Comment:
Could you use `sys.path.insert(1, {gen_input directory} )`; import transLayer as TL rather than including this file?

Reply:
done.

Reply:
I guess this file can be deleted!

Reply:
it is :).